pkg/model/event.proto

```diff
@@ -19,20 +19,18 @@ option go_package = "github.com/pipe-cd/pipe/pkg/model";
 
 import "validate/validate.proto";
 
-message ImageReference {
+message Event {
     // The generated unique identifier.
     string id = 1 [(validate.rules).string.min_len = 1];
-    // The repository name.
-    string repo_name = 2 [(validate.rules).string.min_len = 1];
-    // The image digest.
-    string digest = 3;
-    // The image tags.
-    repeated string tags = 4 [(validate.rules).repeated.min_items = 1];
-    // The ID of the project this image belongs to.
-    string project_id = 5 [(validate.rules).string.min_len = 1];
+    // The name of event.
+    string name = 2 [(validate.rules).string.min_len = 1];
+    // The data of event.
+    string data = 3 [(validate.rules).string.min_len = 3];
+    // The ID of the project this event belongs to.
+    string project_id = 4 [(validate.rules).string.min_len = 1];
```

Comment:
I'm wondering if we really don't have to let Event have repo_id. Currently within a project, all events can be affected all repositories. So it can happen something like below:

Suppose this EventWatcher file is placed at repo1 as well as repo2:
```yaml
apiVersion: pipecd.dev/v1beta1
kind: EventWatcher
spec:
  events:
    - name: app1-image-push
    ...
```

Then you register a new event with the name `app1-image-push`.
```
pipectl event register \
    --address=host.xz:443 \
    --api-key=xxxx
    --event-name=app1-image-push \
    --data=foo/bar:v0.2.0
```

No matter what you intend to update which repository, both repo1 and repo2 will be updated. I figured out we have to prevent it in some way.

Reply:
One way we can solve this with the current design is, we denote users to name the event with the repo prefix like: `repo1/app1-image-push`. We can leave the responsibility to distinguish them to users by doing this.

Reply:
One way to make the event become more usable is that we can add tags/labels field to it.
EventWatcher or anything else can rely on that value.

Reply:
By that way, the event's generality is not reduced. And in the future, we can add another tag/label not even repo id.

Reply:
Totally agree with it. Thanks for the great suggestion, and yeah,  we must not forget keeping up EventWatcher versatile, general and flexible definitely!